lib/aur-sync

```diff
@@ -245,14 +255,13 @@ else
     exit 13
 fi
 
-{ if (( $# )); then
-      printf '%s\n' "$@"
-  fi
+if [[ -v pkgname[@] ]] ; then
+    printf '%s\n' "${pkgname[@]}" >argv
+fi
 
-  if (( update )); then
-      aur vercmp --quiet <db_info
-  fi
-} >argv
```

Comment:
Breaking the grouping emphasize redirection.

Reply:
No reason to do this at all.

Reply:
_revert_